paella-matterhorn/ui/watch.html

```diff
@@ -32,9 +32,10 @@
         <div class="secondary">
           <a href="//www.extension.harvard.edu/help/privacy-policy" id="privacyPolicy">Privacy</a> ::
           <a href="/engage/ui/pubList.html#/tos" id="terms">Terms of Use</a> ::
-          <span id="copyright">&#169;2015 President and Fellows of Harvard College</span>
+          <span id="copyright">&#169;2016 President and Fellows of Harvard College</span>
```

Comment:
This happens before the javascript kicks in. The javascript sets the current year. The idea is to have a header even if javascript is not enabled on client browser. If that is not a requirement, then this could just be removed. The whole header strategy has to be revisited anyway in 2.5 months for the summer, since that requires a default summer header.